modules/accredited_representative_portal/spec/lib/monitor_spec.rb

```diff
@@ -7,6 +7,12 @@
   let(:claim) { create(:saved_claim_benefits_intake) }
   let(:monitor) { described_class.new(claim:) }
 
+  # This removes: SHRINE WARNING: Error occurred when attempting to extract image dimensions:
+  # #<FastImage::UnknownImageType: FastImage::UnknownImageType>
+  before do
+    allow(FastImage).to receive(:size).and_return(nil)
+  end
+
```

Comment:
@stevenjcumming I wonder if we could stub based on file type:
```suggestion
  before do
    allow(FastImage).to receive(:size) do |file|
      file.to_s.end_with?('.pdf') ? nil : FastImage.size.original(file)
    end
  end
```